src/js/components/ToggleButtonGroup/__tests__/ToggleButtonGroup-test.tsx

```diff
@@ -0,0 +1,490 @@
+  });
+
+  test('Should render when options is array of objects with multiple prop', () => {
+    const { container } = render(
+      <Grommet>
+        <ToggleButtonGroup
+          multiple
+          options={[
+            { id: 'onE', label: 'One', value: 'one' },
+            { id: 'twO', label: 'Two', value: 'two' },
+          ]}
+        />
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('Should call onChange in uncontrolled scenarios with multiple prop', () => {
+    const onChange = jest.fn();
+    const { container } = render(
+      <Grommet>
+        <ToggleButtonGroup
+          options={['one', 'two', 'three']}
+          onChange={onChange}
+          multiple
+        />
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+
+    // Simulate a click event on the buttons
+    fireEvent.click(screen.getByRole('button', { name: 'two' }));
+    fireEvent.click(screen.getByRole('button', { name: 'three' }));
+    expect(onChange).toHaveBeenCalledWith(['two', 'three']);
+  });
+
+  test('Should render defaultValue as active with multiple prop', () => {
+    render(
+      <Grommet>
+        <ToggleButtonGroup
+          options={[
+            { label: 'Choice 1', value: 'c1' },
+            { label: 'Choice 2', value: 'c2' },
+            { label: 'Choice 3', value: 'c3' },
+          ]}
+          multiple
+          defaultValue="c2"
+        />
+      </Grommet>,
+    );
+    const activeToggleButton = screen.getByRole('button', {
+      name: /Choice 2/i,
+    });
+    // Assert that the active toggle button exists
+    expect(activeToggleButton).toBeInTheDocument();
+    // Assert that the aria-checked attribute of the active toggle button is true
+    expect(activeToggleButton).toHaveAttribute('aria-checked', 'true');
+  });
+
+  test('Should put tab focus on first button when no value with multiple prop', async () => {
+    const user = userEvent.setup();
+
+    const { container } = render(
+      <Grommet>
+        <ToggleButtonGroup multiple options={['one', 'two', 'three']} />
+      </Grommet>,
+    );
+
+    const focusToggleButton = screen.getByRole('button', {
+      name: /one/i,
+    });
+
+    expect(focusToggleButton).not.toHaveFocus();
+    expect(document.body).toHaveFocus();
+
+    await user.tab();
+    expect(focusToggleButton).toHaveFocus();
+    expect(document.body).not.toHaveFocus();
+
+    expect(focusToggleButton).toHaveStyleRule(
+      'box-shadow',
+      '0 0 2px 2px #6FFFB0',
+      {
+        modifier: ':focus',
+      },
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('Should put tab focus on active button when value with multiple prop', async () => {
+    const user = userEvent.setup();
+
+    const { container } = render(
+      <Grommet>
+        <ToggleButtonGroup
+          options={['one', 'two', 'three']}
+          defaultValue="three"
```

Comment:
Default value should be array. Also, to check behavior can we have `['two', 'three']` then assert that "two" is the one that gets focus

Reply:
Can you double check for all of the `multiple` tests that defaultValue/value are array (even array with single element) vs just a string

Reply:
done